bin/checkout-documentation

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env node
+
+const {docPagesDirectory, versions} = require('../constants');
+const {getPrefixedVersion} = require('../src/lib/versionHelper');
+const {execSync} = require('child_process');
+
+execSync(`rm -rf ${docPagesDirectory}`);
```

Comment:
this looks so dangerous ^^
Do we have to remove things? Can't we just get back the code?

Reply:
ahah I agree with you, I just moved the code from the bash to here, but I'll change this for a git operation :)